pages/common/mpv.md

```diff
@@ -0,0 +1,11 @@
+# mpv
+
+> A media player.
```

Comment:
Perhaps `A video player based on MPlayer` would sound better...

Reply:
It's not just a video player though, it plays audio too (and maybe more, I'm not sure). I could add a line for "play an audio file" just to make that clear.

Reply:
Perhaps something like this would read better?:

```
> A audio and video player.
> Based on MPlayer.
```

Reply:
I've updated this.